community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/MultipleIndexPopulator.java

```diff
@@ -375,13 +377,19 @@ private void populateFromQueueIfAvailable( long currentlyIndexedNodeId )
         {
             try ( MultipleIndexUpdater updater = newPopulatingUpdater( storeView ) )
             {
-                do
+                // TODO the generics of IndexEntryUpdate in combination with that of IndexUpdater make the IndexUpdateProcessor
```

Comment:
do not think that removing generics and adding a long comment that something will not happen is a wright thing to do ever, i think generics should be restored and new code needs to be updated instead.

Reply:
Fixing this in 3.3 will be a bigger thing and a lot of this is undergoing a change in 3.4, so changing anything regarding generics here will only cause a lot of other problems. This should be dealt with in 3.4, after the current work happening in 3.4

Reply:
ok lets do that in 3.4 then only later on